K2Bridge.Tests.UnitTests/KustoDAL/ElasticResponseTests.cs

```diff
@@ -103,7 +103,7 @@ public string MapRowsToHits_WhenMultiWordDifferentCapsCapialLettersInQueryString
         }
 
         [TestCase(ExpectedResult =
-            "{\"responses\":[{\"aggregations\":{\"2\":{\"buckets\":[]}},\"took\":0,\"timed_out\":false,\"_shards\":{\"total\":1,\"successful\":1,\"skipped\":0,\"failed\":0},\"hits\":{\"total\":0,\"max_score\":null,\"hits\":[]},\"status\":200}]}")]
+            "{\"responses\":[{\"aggregations\":{\"2\":{\"buckets\":[]}},\"took\":0,\"timed_out\":false,\"_shards\":{\"total\":1,\"successful\":1,\"skipped\":0,\"failed\":0},\"hits\":{\"total\":{\"relation\":\"eq\",\"value\":0},\"max_score\":null,\"hits\":[]},\"status\":200}],\"took\":0}")]
```

Comment:
is this the actual captured response ?

Reply:
No, I just modified the JSON to match the updated response format ("total" is an object, new "took" element at top-level).